core/contracts/financial-templates/expiring-multiparty/Liquidatable.sol

```diff
@@ -295,6 +295,16 @@ contract Liquidatable is PricelessPositionManager {
             })
         );
 
+        // If this liquidation is a subsequent liquidation on the position, and the liquidation size is larger than minSponsorTokens,
+        // then re-set the liveness. This enables a liquidation against a withdraw request to be "dragged out" if the position is very large.
+        if (
+            positionToLiquidate.withdrawalRequestPassTimestamp > 0 && // this is a subsequent liquidation.
+            positionToLiquidate.withdrawalRequestPassTimestamp < getCurrentTime() && // liveness has not passed yet.
+            maxTokensToLiquidate.isGreaterThan(minSponsorTokens) // above the minimum threshold.
```

Comment:
nit: I think we treat the threshold as a >= threshold rather than a > threshold (look at `create()` so I think this should be `isGreaterThanOrEqualTo`. But it doesn't really matter if it really makes the testing more difficult

Reply:
I agree that we should be consistent here.

Reply:
I have implemented this change. PTAL.